### ISSUE TITLE
Fails if repo doesn't contain a Rails application

### ISSUE THREAD

**Issue by mvz** [mmozuras/pronto-brakeman#7]
Running pronto with pronto-brakeman installed on a non-Rails project (in this case, Reek) results in an error:

```
/home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/brakeman-3.0.3/lib/brakeman/scanner.rb:28:in `initialize': Please supply the path to a Rails application. (Brakeman::NoApplication)
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/brakeman-3.0.3/lib/brakeman.rb:320:in `new'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/brakeman-3.0.3/lib/brakeman.rb:320:in `scan'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/brakeman-3.0.3/lib/brakeman.rb:62:in `run'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/pronto-brakeman-0.4.1/lib/pronto/brakeman.rb:15:in `run'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/pronto-0.4.1/lib/pronto.rb:60:in `block in run_all_runners'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/pronto-0.4.1/lib/pronto.rb:59:in `map'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/pronto-0.4.1/lib/pronto.rb:59:in `run_all_runners'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/pronto-0.4.1/lib/pronto.rb:34:in `run'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/pronto-0.4.1/lib/pronto/cli.rb:52:in `run'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /home/matijs/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/pronto-0.4.1/bin/pronto:6:in `<top (required)>'
	from /home/matijs/.rbenv/versions/2.2.2/bin/pronto:23:in `load'
	from /home/matijs/.rbenv/versions/2.2.2/bin/pronto:23:in `<main>'
```

**Comment by mmozuras** [comment]
@mvz any thoughts on the expected output? :smile:

**Comment by mvz** [comment]
Probably, no output at all from `pronto-brakeman`, alternatively, a notice on STDERR like `Not a Rails app: Skipping brakeman checks.`.

**Comment by mmozuras** [comment]
Decided on "no output" and released as [v0.4.2](https://github.com/mmozuras/pronto-brakeman/releases/tag/v0.4.2).

**Comment by mvz** [comment]
Awesome, thanks!
